### PR TITLE
 artifactory hotfix

### PR DESCRIPTION
**Pull request by PragyeshMishra01**
You can use the following comments to re-trigger PR Checks

- Compile: `trigger compile`
- runAeriformCheck: `trigger AeriformCheck`
- CodeFormat: `trigger codeformat`
- MessageMetadata: `trigger messagecheck`
- Recency: `trigger recency`
- BuildNumberMetadata: `trigger buildnum`
- runDockerizationCheck: `trigger dockerizationcheck`
- runAuthorCheck: `trigger authorcheck`
- Checkstyle: `trigger checkstyle`
- PMD: `trigger pmd`
- TI-bootstrap: `trigger ti0`
- TI-bootstrap1: `trigger ti1`
- TI-bootstrap2: `trigger ti2`
- TI-bootstrap3: `trigger ti3`
- TI-bootstrap4: `trigger ti4`
- FunctionalTest1: `trigger ft1`
- FunctionalTest2: `trigger ft2`
- CodeBaseHash: `trigger codebasehash`

---
This change is [Reviewable](https://reviewable.io/reviews/harness/harness-core/32765)

### CODE REVIEW

**Comment by PragyeshMishra01** [comment]
trigger feature-build

**Review by johannesHarness — DISMISSED** [review]
> Reviewed 1 of 22 files at r1.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/32765)* status: 21 file(s) to review.

File: 127-cd-nextgen-beans/src/main/java/io/harness/serializer/kryo/NGBeanKryoRegistrar.java [hunk 1]
```diff
@@ -60,9 +60,9 @@ public void register(Kryo kryo) {
     kryo.register(DockerArtifactOutcome.class, 8007);
     kryo.register(GcrArtifactOutcome.class, 390006);
     kryo.register(EcrArtifactOutcome.class, 390007);
-    kryo.register(ServerlessAwsLambdaManifestOutcome.class, 390008);
     kryo.register(ServerlessAwsLambdaInfrastructureOutcome.class, 390009);
     kryo.register(ArtifactoryGenericArtifactOutcome.class, 390010);
     kryo.register(AcrArtifactOutcome.class, 390011);
+    kryo.register(ServerlessAwsLambdaManifestOutcome.class, 390012);
```
Comment (johannesHarness) [inline 127-cd-nextgen-beans/src/main/java/io/harness/serializer/kryo/NGBeanKryoRegistrar.java:66]: what's the reason for changing the registered id for this class? won't that break things in prod?

Comment (PragyeshMishra01) [reply 127-cd-nextgen-beans/src/main/java/io/harness/serializer/kryo/NGBeanKryoRegistrar.java:66]: this class is not there in prod yet...initially it had a id:390008 which was not unique so it changed it to 390012 to make it unique...I merged the original pr last week only in which I did registered this class.

Comment (PragyeshMishra01) [reply 127-cd-nextgen-beans/src/main/java/io/harness/serializer/kryo/NGBeanKryoRegistrar.java:66]: this class is not there in prod yet...initially it had a id:390008 which was not unique so it changed it to 390012 to make it unique...I merged the original pr last week only in which I did registered this class.

Comment (johannesHarness) [reply 127-cd-nextgen-beans/src/main/java/io/harness/serializer/kryo/NGBeanKryoRegistrar.java:66]: oh okay, good catch then!

**Review by johannesHarness — APPROVED** [review]
> Reviewed 1 of 13 files at r2.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/32765)* status: 34 file(s) to review.

**Pull request closed without merging**